swagger.yaml

```diff
@@ -2818,38 +2882,73 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
-  operator:
+  monitored-operator:
     type: object
     properties:
+      cluster_id:
+        type: string
+        format: uuid
+        description: The cluster that this operator is associated with.
+        x-go-custom-tag: gorm:"primary_key;foreignkey:Cluster"
+      name:
```

Comment:
So since we are here returning just a unique name I guess that in assisted-installer monitoring part we will hard code specific OLM name/namespace, as it's hardcoded in the operators implementation - meaning that for example for OCS we will fetch `subscription` named `ocs-operator` in namespace `openshift-storage` and it won't be anyhow configurable.

Reply:
Right. Any reason why we would want to support different namespaces? @avishayt

Reply:
we should have namespace, resource_prefix for olm operators part.  controller will be able to monitor anything without those params